crates/oxc_cli/src/lib.rs

```diff
@@ -126,7 +126,7 @@ impl Cli {
         let parser_source_text = source_text.clone();
         let ret = Parser::new(&allocator, &parser_source_text, source_type).parse();
         let result = if ret.errors.is_empty() {
-            let program = allocator.alloc(ret.program);
+            let program = allocator.emplace_no_drop().value(ret.program);
```

Comment:
These `emplace-*` APIs are pretty new to me.

Reply:
`Blink` allows dropping values placed into the allocated memory.
`emplace_*` configure how values are placed.